packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -312,19 +313,30 @@ const SettingsSidebarContainer = ({
                     </div>
                   </div>
                   <div className="my-3 space-y-0.5">
-                    {tab.children?.map((child, index) => (
-                      <VerticalTabItem
-                        key={child.href}
-                        name={t(child.name)}
-                        isExternalLink={child.isExternalLink}
-                        href={child.href || "/"}
-                        textClassNames="px-3 text-emphasis font-medium text-sm"
-                        className={`my-0.5 me-5 h-7 ${
-                          tab.children && index === tab.children?.length - 1 && "!mb-3"
-                        }`}
-                        disableChevron
-                      />
-                    ))}
+                    {tab.children?.map((child, index) => {
+                      if (tab.href === "/settings/organizations" && !isOrgAdminOrOwner) {
+                        // Only show profile and members for user role.
+                        if (!["profile", "members"].includes(child.name)) {
+                          return null;
+                        }
+                        if (child.name === "members") {
+                          bottomMargin = true;
+                        }
+                      }
```

Comment:
instead of this can you make changes inside useTabs() hook and remove the tabs that are not required for 'USER' role.

Reply:
you won't need bottomMargin this way

Reply:
Yeah that's better, will make changes.